djstripe/models.py

```diff
@@ -29,9 +28,10 @@
 from django.utils import dateformat, six, timezone
 from django.utils.encoding import python_2_unicode_compatible, smart_text
 from django.utils.functional import cached_property
+import stripe
```

Comment:
my IDE's isort settings are probably messed up. lmk if I need to fix this.

Reply:
Just make sure you're running the latest version of isort, which should pick up on setup.cfg